app/views/calculator/payment_summary.scala.html

```diff
@@ -78,8 +78,8 @@ <h1 class="govuk-heading-xl">@messages("ssttp.calculator.form.payment_summary.ti
             @messages("ssttp.calculator.form.payment_summary.amount.remaining")
           </dt>
           <dd class="govuk-summary-list__value">
-            @currency(debits.map(_.amount).sum - initialPayment)
-              <p class="small govuk-!-font-weight-regular govuk-!-margin-top-1">@messages("ssttp.calculator.form.payment_summary.amount.remaining.hint")</p>
```

Comment:
Is there a reason for not having a <p> element here and in line 63?

Reply:
Jenkins was reporting it as an error in those 2 places.